yarn.lock

```diff
@@ -10632,7 +10647,7 @@ react-source-render@2.0.0-beta.6:
     prop-types "^15.6.2"
     react-is "^16.8.1"
 
-react-test-renderer@^16.0.0, react-test-renderer@^16.0.0-0:
+react-test-renderer@^16.0.0-0:
```

Comment:
same here - is there any reason for us to have these two (I mean, _non-addressable_ reason:)

Reply:
Deduped it